umake/tools.py

```diff
@@ -337,6 +337,15 @@ def create_launcher(desktop_filename, content):
         gsettings.set_strv("favorites", launcher_list)
 
 
+def add_exec_link(exec_path, destination_name):
```

Comment:
yeah, try with @patch() or mock
(look at some of our unit tests examples using mock/patch overriding either a some of the tools.py method).